motr/examples/example1.c

```diff
@@ -36,7 +36,7 @@
  * Please change the configuration according to you development environment.
  *
  * How to run:
- * LD_LIBRARY_PATH=/work/cortx-motr/motr/.libs/                              \
+ * export LD_LIBRARY_PATH=/work/cortx-motr/motr/.libs/                              \
```

Comment:
Please note the last char in this line, the back slash.
That means, this line, the next, and the next next line, should be put in a single line in shell.

So, you don't need to "export LD_LIBRARY_PATH", because it is done for this run.

Reply:
Got it. Sorry about that